src/nft/components/explore/CarouselCard.tsx

```diff
@@ -12,14 +12,20 @@ const CarouselCardContainer = styled.div`
   flex-direction: column;
   gap: 20px;
   background-color: ${({ theme }) => theme.backgroundSurface};
+  border: 1px solid ${({ theme }) => theme.backgroundOutline};
   border-radius: 20px;
-  outline: 1px solid ${({ theme }) => theme.backgroundOutline};
+  overflow: hidden;
+  height: 100%;
+`
+const CarouselCardBorder = styled.div`
   width: 100%;
+  position: relative;
+  border-radius: 22px;
   cursor: pointer;
-  overflow: hidden;
+  border: 2px solid transparent;
```

Comment:
I know it didn't have this before, but we should add a transition on border (guessing theme.transition.duration.fast or .medium and theme.transition.timing.inOut will feel best)

Reply:
Okay, let's add fast/inOut and ship it! Not a launch blocker, I guess we can always revisit later

Reply:
Generally i agree that default transition feels a bit rough